packages/guideline-blocks-settings/src/components/RichTextEditor/plugins/shared/LinkSelector/LinkSelector.tsx

```diff
@@ -49,7 +49,7 @@ export const LinkSelector = ({ appBridge, url, onUrlChange }: LinkSelectorProps)
             >
                 Internal link
             </Button>
-            <Modal zIndex={200} onClose={() => closeLinkTree()} isOpen={isLinkTreeOpen} isDismissable>
+            <Modal zIndex={200000} onClose={() => closeLinkTree()} isOpen={isLinkTreeOpen} isDismissable>
```

Comment:
Smelly code

Reply:
Thanks, fixed to 1001. The link selector modal has a z index of 1000, that's why we need it higher than that.